packages/magento-product/components/ProductPageCategoryBreadcrumb/ProductPageCategoryBreadcrumb.tsx

```diff
@@ -0,0 +1,24 @@
+import { Breadcrumbs, Link, Typography } from '@mui/material'
+import { ProductPageCategoryBreadcrumbFragment } from './ProductPageCategoryBreadcrumb.gql'
+
+export function ProductPageCategoryBreadcrumb(props: ProductPageCategoryBreadcrumbFragment) {
+  const { breadcrumbs, name } = props
+  return (
+    <Breadcrumbs>
+      <Link underline='hover' color='inherit' href='/'>
+        Home
+      </Link>
+      {breadcrumbs?.map((mapped_category, i) => (
+        <Link
+          underline='hover'
+          key={mapped_category?.category_uid}
+          color='inherit'
+          href={`/${mapped_category?.category_url_path}`}
+        >
+          {`${mapped_category?.category_name}`}
+        </Link>
+      ))}
+      <Typography color='text.primary'>{name}</Typography>
+    </Breadcrumbs>
+  )
+}
```

Comment:
Functionality of this component is only specific to the category and shouldn't be in the magento-product package. Should be moved to `packages/magento-category/components/CategoryBreadcrumb/CategoryBreadcrumb.tsx`